tests/snapshots/framework-detection.test.js.md

```diff
@@ -223,3 +223,16 @@ Generated by [AVA](https://avajs.dev).
        └──────────────────────────────────────────────────┘␊
     ␊
     `
+
+## should pass framework-info env to framework sub process
+
+> Snapshot 1
+
+    `◈ Netlify Dev ◈␊
+    ◈ Starting Netlify Dev with Gatsby␊
+    ␊
+    > develop␊
+    > echo $GATSBY_LOGGER␊
```

Comment:
This will fail on Windows since we use `%GATSBY_LOGGER%` instead. 
See https://github.com/netlify/cli/pull/3194/files#r690296379